Source/Core/VideoCommon/Assets/CustomAssetLoader.cpp

```diff
@@ -52,9 +52,8 @@ void CustomAssetLoader::Init()
       {
         if (m_max_memory_available >= m_total_bytes_loaded + ptr->GetByteSizeInMemory())
         {
+          std::lock_guard lk(m_asset_load_lock);
```

Comment:
There's two calls to `GetByteSizeInMemory()` here that both lock the info lock, you can and probably should store that result.

Reply:
Probably move that `std::lock_guard lk(m_asset_load_lock);` outside the `if`, too...